docs/agent/config.md

```diff
@@ -66,6 +66,16 @@ in any way.
 || `syslog_key`  | Syslog configuration client private key for TLS client validation |
 
 
+## Integrations init_config section
+
+In addition to integration-specific options, the agent supports the following
+options in the `init_config` section:
+
+* `min_collection_interval`: number of seconds between two runs of a given
```

Comment:
Could we explain quickly when/why people should use those option ? I don't want every customer to set the `min_collection_interval` to 1s.